### DISCUSSION TITLE
Github actions timeout

### DISCUSSION THREAD

**Discussion by benel** [community/community#26595] [Actions]
<p>Hello everyone!</p>
<p>Two months ago I switched from Travis CI to GitHub actions to run my acceptance tests (with Cucumber and Capybara).<br>
Whereas tests were “flaky” on the former, they became completely predictable since the switch (for two months).</p>
<p>Last evening, they started to fail (even with retries). With those kind of messages that I interpret as timeouts:</p>
<blockquote>
<p>Request to <a href="http://fv-az186-255:3000/item/Vitraux" rel="noopener nofollow ugc">http://fv-az186-255:3000/item/Vitraux</a> - Bénel/D65963E34F30C462B600A7AF6DDC3F0B73F9 reached server, but there are still pending connections: <a href="http://fv-az186-255:3000/item/Vitraux%20-%20B%C3%A9nel/D65963E34F30C462B600A7AF6DDC3F0B73F9" rel="noopener nofollow ugc">http://fv-az186-255:3000/item/Vitraux%20-%20Bénel/D65963E34F30C462B600A7AF6DDC3F0B73F9</a> (Ferrum::StatusError)</p>
</blockquote>
<p>When I “re-run” the job, sometimes it succeeds, but most of the time it doesn’t. They are not predictable anymore :cry:</p>
<p>Could it be related:</p>
<ul>
<li>to a change in the infrastructure (but I use mainly dockerized apps),</li>
<li>to a high load on GH Actions (I’ve read about the cryptominers attack one week ago… but GH status tells everything is OK right now),</li>
<li>to an error of mine (but that does not explain why it succeeds from time to time)?</li>
</ul>
<p>I really hope to find a solution before tomorrow: my students will discover continuous integration, BDD and TDD, and I wouldn’t like them to have a bad opinion of it because of this problem.</p>
<p>Best regards.</p>

**Comment by lhotari** [comment, marked as answer]
<p>Please check <a href="https://github.community/t/github-action-and-oserror-errno-99-cannot-assign-requested-address/173973/4" class="inline-onebox">Github Action and OSError: [Errno 99] Cannot assign requested address - #4 by lhotari</a> for a solution.</p>

**Comment by benel** [comment]
<p>Thank you so much <a class="mention" href="/lhotari">@lhotari</a>!<br>
You saved my tutorial for tomorrow!!! :slight_smile:</p>
